Cache post run results in file name too long error on Windows

<img width="2085" alt="image" src="https://user-images.githubusercontent.com/5013932/92210666-5e7f8f80-eeb9-11ea-8f68-6d12f7f09537.png">

https://github.com/prisma/prisma-client-go/runs/1070606307

**Describe the bug**
windows action results in "file name too long" error

**To Reproduce**
run a windows action

**Expected behavior**
succeed

**Debug logs**

```
"C:\Program Files\Git\bin\sh.exe" -c "docker save '57e8659cfbe3' '57e8659cfbe3' 'b0a4bbcf733c' 'aa010ec86e43' '3e2306a9e69a' '170e5eed2561' '1fcb9726982c' '4ad344606eba' '363820071cb5' '363820071cb5' '07f56b823f92' '3096552b55c7' 'eb1ae8874308' '8fc9e243f6ac' '076b819af1b1' 'be1b4ed60961' 'c969c043a907' 'b6fb5fc8fc3f' '262eb6b25268' '1fcb9726982c' '4ad344606eba' '4ad344606eba' '4ad344606eba' 'integration:latest' '57e8659cfbe3' 'b0a4bbcf733c' 'aa010ec86e43' '3e2306a9e69a' '170e5eed2561' '1fcb9726982c' '4ad344606eba' 'golang:1.13' '4ad344606eba' | tar xf - -C ."
3
tar: f84e061c1b08d5b5c406eb8e6b392d60db53a246f24dad44cbe287c827acf2e4/layer.tar: Cannot create symlink to '..\\e6d34c02c003006582cb6abd097c3c1bbd3c583d2800e640ff04cdf645b5d615\\layer.tar': File name too long
4
tar: f4fdc24f3d20f364752816cb7dc2bb21e0ba6367cbd6cfe846422c67f3066ddf/layer.tar: Cannot create symlink to '..\\363a20c3cc7ea6f5c97f3907383603ec140a4bfb166bea1dc6e385fbe840f800\\layer.tar': File name too long
5
tar: c521749a4bd1422c0e9dcc5c2d8bc9a06f9d075aebbc800c0a3abec8ca156c1a/layer.tar: Cannot create symlink to '..\\814c4412fd8c41b88e7fc1d05da67d24103c50c24f3a53be8c2fd0329ff98cb2\\layer.tar': File name too long
6
tar: ae587939341bfafb404e90b0642c0df2867aaebe3a3965cdf0754fc65eb2783a/layer.tar: Cannot create symlink to '..\\2f7914e58cf2ed2ba7bc7672908fd8602648c25089a2efc6a46c1bfb939ec0ac\\layer.tar': File name too long
7
tar: 6ebbd9e839ed2e5e6ac9b2d853363e0119539d1ca4c39e6741a0289bf0bb046d/layer.tar: Cannot create symlink to '..\\8eeffc8140192a406c195601ca9a8143df661c0f0c136733cff5a7b837cbc0be\\layer.tar': File name too long
8
tar: 6d377303e4c7ad08d49b64f3d99f2ddf8e08190dd8cb1fdb7562e0001c372463/layer.tar: Cannot create symlink to '..\\997f17eb7110b3f397f5273ba11c447e85ad1ed91a208ad1fdc17db386e63c18\\layer.tar': File name too long
9
tar: 6c5b1bd5e2f5522aba7662223d13a98bf1c34785a260f248cc5d77e7370a0d59/layer.tar: Cannot create symlink to '..\\2705ca6221af391723cfab4bf3540a5bd229f552190a67e9e168e05b6fe3b1ee\\layer.tar': File name too long
10
tar: 655c054642cc83e59a3b894e5b4254deec6c04d4b757d5057f3d029788e8ded8/layer.tar: Cannot create symlink to '..\\4c1d96bc66f2acb88bc40795a303f317daff2b2b2f617ba98b1d2e1a762dffe7\\layer.tar': File name too long
11
tar: 5a31b519666696f0ceb4f4e67aeb1a9dffc6f41d2140978cfdc58e578f64dea9/layer.tar: Cannot create symlink to '..\\dca8603104db128a47f183ca638d580d871b5fc107143bd3ffe19130de686211\\layer.tar': File name too long
12
tar: 55b0db4f36d5e6f5f58e5fbb0a1cc9e0b42f831d77a74a65942846a8e3d17311/layer.tar: Cannot create symlink to '..\\e49be28a0162855ec9267c1a6b0b28b8131bd8eb21fab7b737474393677a96ee\\layer.tar': File name too long
13
tar: 4e9c6bee90ac5afe0a3f126352aa8bd2ac6a5c1c27ec537e9a9e856b73515dc2/layer.tar: Cannot create symlink to '..\\8507ab3b7f9450cebfd7f5a8909141ba84af4760b25be71c921fb32846cab31a\\layer.tar': File name too long
14
tar: 2b59e1cc041594149a367b0eb4fce42abca8eecc2b30395f685a095baa2f5b93/layer.tar: Cannot create symlink to '..\\2dc7aa7ba3c7d0a39552250998cfdc42ab131f9eb2a8ef78d506496b55231627\\layer.tar': File name too long
15
tar: 15b21088ba8ab0cb2e14c8f6aa29889ddb88f6eea05e9b68583f0d546db5ea94/layer.tar: Cannot create symlink to '..\\78c82bc0408ad7dc44b6eb1213747bee9bfe7c3696fb2713a34389f510d0ce52\\layer.tar': File name too long
16
tar: 070eebd009d890e2855e5bb0f4f4e7024fb34cd6d9739c04412c7eaab5a81f7c/layer.tar: Cannot create symlink to '..\\473dbcaaf6d6608092934f70e10461a76ca73a97b13ebdc3def6b22992bd0e5b\\layer.tar': File name too long
17
tar: 0382a3a452c02bb011218a9c3b3d99491f888b75c353260896a9626b77e62d20/layer.tar: Cannot create symlink to '..\\235f2f7108ecde4a6302fab5ee81f8c9cc5e4afa1dddc3cdc46d7eed4db164bd\\layer.tar': File name too long
18
tar: Exiting with failure status due to previous errors
19
Error: The process 'C:\Program Files\Git\bin\sh.exe' failed with exit code 2
20
    at ExecState._setResult (D:\a\_actions\satackey\action-docker-layer-caching\v0.0.8\dist\ttsc-dist\post.js\index.js:1300:25)
21
    at ExecState.CheckComplete (D:\a\_actions\satackey\action-docker-layer-caching\v0.0.8\dist\ttsc-dist\post.js\index.js:1283:18)
22
##[error]Error: The process 'C:\Program Files\Git\bin\sh.exe' failed with exit code 2
23
    at ChildProcess.<anonymous> (D:\a\_actions\satackey\action-docker-layer-caching\v0.0.8\dist\ttsc-dist\post.js\index.js:1183:27)
24
    at ChildProcess.emit (events.js:210:5)
25
    at maybeClose (internal/child_process.js:1021:16)
26
    at Socket.<anonymous> (internal/child_process.js:430:11)
27
    at Socket.emit (events.js:210:5)
28
    at Pipe.<anonymous> (net.js:659:12)
```

**Runner Environment (please complete the following information):**
 - OS: [e.g. windows-latest]
 - Action version: [e.g. v0.0.8]

**Additional context**

--- a/src/LayerCache.ts
+++ b/src/LayerCache.ts
@@ -14,7 +14,7 @@ class LayerCache {
   ids: string[] = []
   unformattedSaveKey: string = ''
   restoredRootKey: string = ''
-  imagesDir: string = path.resolve(`${__dirname}/../.action-docker-layer-caching-docker_images`)
+  imagesDir: string = path.resolve(`${__dirname}/../.adlc`)
   enabledParallel = true
   concurrency: number = 4
 
@@ -178,8 +178,6 @@ class LayerCache {
   }
 
   private async restoreLayers(): Promise<boolean> {
-
-    
     const pool = new PromisePool(this.concurrency)
     const tasks = (await this.getLayerIds()).map(
       layerId => pool.open(() => this.restoreSingleLayerBy(layerId))

